libs/s25main/desktops/dskHostGame.cpp

```diff
@@ -1019,3 +1036,22 @@ void dskHostGame::LC_Chat(const std::string& player, const std::string& text)
             lobbyChatTabAnimId = tab->GetAnimationManager().addAnimation(new BlinkButtonAnim(bt));
     }
 }
+
+bool dskHostGame::checkOptions()
+{
+    if(forceOptions)
+        return true;
+    const GlobalGameSettings& ggs = gameLobby->getSettings();
+    if(ggs.getSelection(AddonId::PEACEFULMODE)
+       && (ggs.objective == GO_CONQUER3_4 || ggs.objective == GO_TOTALDOMINATION))
+    {
+        WINDOWMANAGER.Show(std::make_unique<iwMsgbox>(
+          _("peaceful Mode"),
```

Comment:
```suggestion
          _("Peaceful mode"),
```
It is a title, so capitalize the first word